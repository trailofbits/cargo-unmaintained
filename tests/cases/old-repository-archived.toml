path = "fixtures/old-repository-archived"
