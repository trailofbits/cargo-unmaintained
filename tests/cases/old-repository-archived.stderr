Scanning 3 packages and their dependencies (pass --verbose for more information)
